helm-charts/FATE/templates/pulsar.yaml

```diff
@@ -125,7 +125,7 @@ data:
     backlogQuotaCheckIntervalInSeconds=60
     
     # Default per-topic backlog quota limit
-    backlogQuotaDefaultLimitGB=10
+    backlogQuotaDefaultLimitGB=-1
```

Comment:
What is the effect of this change?

Reply:
Unlimit the backlog quota may cause the storage not enough problem. Suggest pass through this setting to user, and set -1 as the default value.

Reply:
Yes, unlimited backlog quota, but  brokerDeleteInactiveTopicsFrequencySeconds will delete it when pulsar recognize the topic as inactive.

Reply:
I am not sure if I understand it right, but the space needed for the backlog = messages number * message size * delete frequency. So if the storage is less than accumulated messages in 36000 seconds (we set in default), we will still run out of storage. Right?